manifests/init.pp

```diff
@@ -439,6 +441,14 @@
     notify  => Service['snmpd'],
   }
 
+  # Debian 9 use systemd
+  if ( $facts['os']['name'] == 'Debian' and versioncmp($facts['os']['release']['major'], '9') >= 0 ) {
```

Comment:
This will also match Debian 10. Do you have to make the same change you made to data/os/Debian/9.yaml to data/os/Debian/10.yaml ??

Reply:
@antondollmaier Are you able to help fix this PR and confirm it works on Debian 10?

Reply:
I updated data/os/Debian/10.yaml according to default value found on docker image. will see how it goes.